test/torchaudio_unittest/functional/autograd_impl.py

```diff
@@ -337,8 +312,8 @@ def test_mvdr_weights_rtf_with_tensor(self):
             (3, False),
         ]
     )
-    def test_rtf_power(self, n_iter, diagonal_loading):
-        torch.random.manual_seed(2434)
+
+    def test_rtf_power(self, n_iter):
```

Comment:
The `diagonal_loading` argument should not be removed.

Reply:
Should it be fine to remove the manual_seed however?

Reply:
yes manual seed should be removed here, as it's set in the setup function